src/raymath.h

```diff
@@ -2549,9 +2549,13 @@ RMAPI void MatrixDecompose(Matrix mat, Vector3 *translation, Quaternion *rotatio
 
     // Extract scale
     const float det = a*A + b*B + c*C;
-    float scalex = Vector3Length((Vector3){ a, b, c });
-    float scaley = Vector3Length((Vector3){ d, e, f });
-    float scalez = Vector3Length((Vector3){ g, h, i });
+	Vector3 abc = { a, b, c };
```

Comment:
Please, could you review the TAB by 4 spaces?

Reply:
fixed, sorry, did this on a different computer than I normally use.